client/lib/domains/index.js

```diff
@@ -158,6 +161,18 @@ function hasMappedDomain( domains ) {
 	return getMappedDomains( domains ).length > 0;
 }
 
+function isOpenSrsDomain( domain ) {
+	return domainEndsWithTld( domain, [ '.live' ] );
+}
+
+function isOpenHrsDomain( domain ) {
+	return domainEndsWithTld( domain, [ '.wales' ] );
+}
+
+function domainEndsWithTld( domain, tlds ) {
+	return some( tlds, ( tld ) => endsWith( domain, tld ) );
+}
```

Comment:
Too much abstraction? I don't think `domainEndsWithTld` is necessary, as it's a one liner of an already clearly purposed function. How about extracting `HRS_DOMAINS` and `SRS_DOMAINS` as constants (and potentially getting this info from server in the future maybe) and calling them like `some( HRS_DOMAINS, partial( endsWith, domain )`?

Arrow function works as well if you're not a fan of partials